src/lib/common/errorMessages.h

```diff
@@ -26,7 +26,10 @@
 * Author: Orion dev team
 */
 
-#define MORE_MATCHING_ENT "More than one matching entity. Please refine your query"
-#define INVAL_CHAR_URI    "invalid character in URI"
+#define MORE_MATCHING_ENT   "More than one matching entity. Please refine your query"
+#define INVAL_CHAR_URI      "invalid character in URI"
+#define EMPTY_ENTITY_ID     "entity id length: 0, min length supported: 1"
```

Comment:
What about if min length change in the future? I'd suggest to use MIN_ID_LEN to compose these messages, e.g:

```
#define EMPTY_ENTITY_ID     "entity id length: 0, min length supported: " MIX_ID_LEN
```

(Actually I'm not sure if this is the right way of concatenating #define constants)